app/lib/api.js

```diff
@@ -1,6 +1,6 @@
 export function getStrapiURL(path = "") {
   return `${
-    process.env.NEXT_PUBLIC_STRAPI_API_URL || "http://127.0.0.1:1337"
+    process.env.NEXT_PUBLIC_STRAPI_API_URL || "http://172.28.0.1:1337"
```

Comment:
Do we need to change this default value? Since `127.0.0.1` is common to almost all local workspaces. But the `172.28.0.1` may be local to only your machine.
I would suggest writing and storing this IP address as a `.env` var `NEXT_PUBLIC_STRAPI_API_URL="http://{ip_address}:1337"` and I think you can fetch the IP address using some docker inspect command.

Please add it as an optional script, it should run for anyone who is going to use the NextJS docker image, for a manual start (not using a docker image) the script should not run.

Thank you!

Reply:
Yeah, we need them as default values. It's not the IP address of the device instead it's the IP address of the gateway between the Docker host and the bridge network.  We can set any value unless it is a subnet value of 172.18.0.0/16 . You can find the whole documentation here - https://docs.docker.com/network/network-tutorial-standalone/#:~:text=Near%20the%20top%2C%20information%20about,0.2%20for%20alpine1%20and%20172.17.

AFAIK It will be better if we keep this as the default value.

Reply:
Thanks for sharing the resources; still, it is configurable, right?

So we need to handle those cases; IMO, it will not be a good developer experience if the new contributor has to get and set these values. Thoughts?

Reply:
Thank you for the review. 
AFAIK, I don't think we need to handle cases since the value we have defined is a subnet value of 172.18.0.0/16 so it will work in all the cases.